tests/test_pipreqs.py

```diff
@@ -322,6 +324,30 @@ def test_clean_with_imports_to_clean(self):
             data = f.read().lower()
             self.assertTrue(cleaned_module not in data)
 
+    def test_output_requirements(self):
+        """
+        Test --print parameter
+        It should print to stdout the same content as requeriments.txt 
+        """
+
+        capturedOutput = io.StringIO()          
+        sys.stdout = capturedOutput
+
+        pipreqs.init(
+            {'<path>': self.project, '--savepath': None, '--print': True,
+            '--use-local': None, '--force': None, '--proxy': None,
+            '--pypi-server': None, '--diff': None, '--clean': None,
+            '--mode': None}
+            )
+        pipreqs.init({'<path>': self.project, '--savepath': None, '--print': False,
+                      '--use-local': None, '--force': True, '--proxy':None, '--pypi-server':None,
+                      '--diff': None, '--clean': None, '--mode': None})
+        
+        with open(self.requirements_path, "r") as f:
+            file_content = f.read().lower()
```

Comment:
remove lower, should be unnecessary

Reply:
Solved on [321cb8b](https://github.com/bndr/pipreqs/pull/395/commits/321cb8b45ebc340d3d5395c4fe236169319bbe64).

Reply:
thanks @fredgrub ! Last thing about this lower issue, could you reword the commit?

Give some more context about the issue and its solution, if someone reads "remove lower" as a commit message they wont have any idea as to what the commit does or why it was necessary, they will need to go to the commit changes and try to decipher what it means and why it was necessary.

There is no need to write a whole essay but some additional context and it information would be good

Reply:
sorry for being a pain in the ass haha but #381 #382 raise a fair point... Do you have any thoughts about it? maybe it would be better to just drop this "lower" commit from this PR and create a new task for issue #381

Reply:
I think that the issue of case sensitivity should be moved to a separate pull request. I just dropped the "remove lower" commit.